.github/scripts/check-for-go-mod-changes.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -ue
+
+for DIRECTORY in . ./pkg/apis ./e2e/testenv/infra; do
```

Comment:
What about `find`-ing `go.mod` files in the repository and run this on every location?

Reply:
Hmm not worth it then, thanks!

Reply:
Maybe something like this?
`find . -name 'go.mod' -execdir go mod tidy \;`